homeassistant/components/ialarm_xr/alarm_control_panel.py

```diff
@@ -48,7 +56,19 @@ def __init__(self, coordinator: IAlarmXRDataUpdateCoordinator) -> None:
     @property
     def state(self) -> str | None:
         """Return the state of the device."""
-        return self.coordinator.state
+
+        state = STATE_UNKNOWN
```

Comment:
We use `None` to represent unknown state in the platforms. `None` will be converted by the base entity class to `STATE_UNKNOWN`.